src/main/java/com/google/jenkins/plugins/credentials/oauth/GoogleRobotPrivateKeyCredentials.java

```diff
@@ -50,22 +51,6 @@ public final class GoogleRobotPrivateKeyCredentials extends GoogleRobotCredentia
   @Deprecated private transient String secretsFile;
   @Deprecated private transient String p12File;
 
-  /**
-   * Construct a set of service account credentials.
-   *
-   * @param projectId The project id associated with this service account
-   * @param serviceAccountConfig The ServiceAccountConfig to use
-   * @param module The module for instantiating dependent objects, or null.
-   */
-  public GoogleRobotPrivateKeyCredentials(
```

Comment:
What I think, are we sure that this constructor (and others we deleted in this PR in multiple classes) is not used anywhere in the Jenkins ecosystem? Maybe we should play safe and deprecate it and set scope=GLOBAL by default? WDYT?

Reply:
returned them and marked them as deprecated